src/services/cube-builder.ts

```diff
@@ -596,8 +590,10 @@ async function loadFactTablesWithUpdates(
     );
 
     if (dataTablePresent.length === 0) {
-      logger.warn('Data table not loaded in to data_tables schema.  Loading data table from blob storage.');
-      await loadFileIntoDataTablesSchema(dataset, dataTable);
+      logger.error(
+        `Data table ${dataTable.id} not loaded in to data_tables schema.  Loading data table from blob storage.`
+      );
```

Comment:
Same here, is the "Loading data table from blob storage" message still valid?

Reply:
We don't care at this point... The whole cube builder is being changed in about 2 PRs time.